addon/instance-initializers/cloudinary-config.js

```diff
@@ -1,4 +1,9 @@
-import Ember from 'ember';
+/*global Ember */
+//import Ember from 'ember';
+export default {
```

Comment:
Please remove and revert this file, stay with Ember addon conventions. Was this causing a problem for you? Additionally, you cannot use Ember (line 17) if it is not imported. Do not make Ember global.